apps/designer/app/designer/features/sidebar-left/panels/asset-manager/asset-tooltip.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+import { Button, Tooltip } from "@webstudio-is/design-system";
+import { GearIcon } from "@webstudio-is/icons";
+import type { Asset } from "@webstudio-is/prisma-client";
+import { AssetInfo } from "./asset-info";
+
+export const AssetTooltip = ({
+  asset,
+  onDelete,
+}: {
+  asset: Asset;
+  onDelete: () => void;
+}) => {
+  const [isTooltipOpen, setTooltipOpen] = useState(false);
+  const closeTooltip = () => setTooltipOpen(false);
+
+  return (
+    <Tooltip
+      open={isTooltipOpen}
+      multiline
+      onEscapeKeyDown={closeTooltip}
+      onPointerDownOutside={closeTooltip}
+      css={{ width: 240, maxWidth: 240, padding: 0, paddingBottom: "$2" }}
```

Comment:
I think you wanna use PANEL_WIDTH constant

Reply:
Done!